python/lsst/pipe/base/graphBuilder.py

```diff
@@ -677,8 +684,13 @@ def fillQuanta(self, registry, inputCollections, *, skipExisting=True):
                                 expand=True,
                             )
                         )
+                    if refs == []:
```

Comment:
Should not compare equality to an empty list, either check the length equal to zero, or just do `if refs` as lists are truthy.